gta/dff_importer.py

```diff
@@ -68,6 +68,10 @@ class dff_importer:
     use_mat_split      = False
     remove_doubles     = False
     group_materials    = False
+    initial_object_position = mathutils.Vector((0,0,0))
+    initial_object_rotation = mathutils.Quaternion((1,0,0,0))
+    initial_object_scale = mathutils.Vector((1,1,1))
```

Comment:
This is a quick and dirty position and rotation assignment. I realise I'll have to add the existing position and rotation to my assigned values, now I'm just overwriting them.

But that aside, you originally suggested that:

> Then it should make new instances of the loaded models (possibly through Linked Duplicate function) and move them to the coordinates and other options set by the IPL file. This is done to not re-load the same file over and over again.

I could use some help with this, I don't really know what you meant by it or how to do it.
Or maybe what I have now is good enough?

The import process is quite slow, maybe your way of creating objects would speed things up a little?